cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -250,3 +261,110 @@ def validate_schema_completeness(self) -> Self:
+    def __init__(self, information: InformationRules):
+        self.information = information
+
+    def as_domain_rules(self) -> DomainRules:
+        raise NotImplementedError("DomainRules not implemented yet")
+
+    def as_dms_architect_rules(self) -> "DMSRules":
+        from .dms_architect_rules import DMSContainer, DMSMetadata, DMSProperty, DMSRules, DMSView
+
+        info_metadata = self.information.metadata
+
+        metadata = DMSMetadata(
+            schema_=info_metadata.schema_,
+            space=info_metadata.prefix,
+            version=info_metadata.version,
+            external_id=info_metadata.name.replace(" ", "_").lower(),
+            contributor=info_metadata.contributor,
+        )
+
+        properties_by_class: dict[str, list[DMSProperty]] = defaultdict(list)
+        for prop in self.information.properties:
+            properties_by_class[prop.class_].append(self._as_dms_property(prop))
+
+        views: list[DMSView] = [
+            DMSView(
+                class_=cls_.class_,
+                view=ViewEntity(prefix=info_metadata.prefix, suffix=cls_.class_),
+                description=cls_.description,
+                implements=[
+                    ViewEntity(prefix=parent.prefix, suffix=parent.suffix, version=parent.version)
+                    for parent in cls_.parent or []
+                ],
+            )
+            for cls_ in self.information.classes
+        ]
+
+        containers: list[DMSContainer] = []
+        classes_without_properties: set[str] = set()
+        for class_ in self.information.classes:
+            properties: list[DMSProperty] = properties_by_class.get(class_.class_, [])
+            if not properties or all(
+                isinstance(prop.value_type, ViewEntity) and not prop.value_type != "direct" for prop in properties
+            ):
+                classes_without_properties.add(class_.class_)
+                continue
+
+            containers.append(
+                DMSContainer(
+                    class_=class_.class_,
+                    container=ContainerEntity(prefix=info_metadata.prefix, suffix=class_.class_),
+                    description=class_.description,
+                    constraint=[
+                        ContainerEntity(
+                            prefix=parent.prefix,
+                            suffix=parent.suffix,
+                        )
+                        for parent in class_.parent or []
+                        if parent.id not in classes_without_properties
+                    ]
+                    or None,
+                )
+            )
+
+        return DMSRules(
+            metadata=metadata,
+            properties=SheetList[DMSProperty](
+                data=[prop for prop_set in properties_by_class.values() for prop in prop_set]
+            ),
+            views=SheetList[DMSView](data=views),
+            containers=SheetList[DMSContainer](data=containers),
+        )
+
+    @staticmethod
```

Comment:
I think that we have been opting for classmethods instead of static methods before.

Reply:
Yes, I am a bit back and fourth on that.